tests/unit/test_config.py

```diff
@@ -67,6 +67,14 @@ def test_get_properties(self):
                                 'd\\=': '4', 'e\\:': '5', 'f': '==6',
                                 'g': '= 7', 'h': ':8', 'i': '9', 'j': '${ENVIRONMENT_VARIABLE}'})
 
+    @patch('prestoadmin.config.os.path.expandvars', return_value='environment_variable_value')
+    def test_expending_environment_variable(self, environment_variables):
+        config_file = os.path.join(DIR, 'resources', 'environment_variable.properties')
+        conf = config.get_conf_from_properties_file(config_file)
+        self.assertTrue(environment_variables.called)
+        self.assertEqual({'a': 'environment_variable_value'}, conf)
```

Comment:
can we have an real test that is using actual environment variable, to see the configuration with expanded environment variables.

Reply:
Sorry for the late reply. I will work on this as well as automation failures.